packages/cli/src/commands/setup/auth/templates/azureActiveDirectory.auth.ts.template

```diff
@@ -1,32 +1,48 @@
-import { parseJWT } from '@redwoodjs/api'
+import { parseJWT, Decoded } from '@redwoodjs/api'
```

Comment:
If you don't have import type separately, I believe it isn't removed in JS templates

Reply:
I just tried. It does remove `Decoded` in JS projects

![image](https://user-images.githubusercontent.com/30793/188153002-12d9f1d7-c9e7-4eb4-9e26-6bd3a9d5121d.png)